contrib/udfs/README.md

```diff
@@ -436,3 +436,11 @@ The functions are:
 
 [1]: https://github.com/target/huntlib
 
+
+# DNS Functions
```

Comment:
It would be nice to mention that the JRE caches DNS records for their TTL which should mean that these functions can scale to big datasets if the number of distinct domains that need to be looked up is not big.